Webpack loader for .jsx issue

* Using 0.6 Kolibri and 0.7 perseus branch.
* Webpack is unable to find the appropriate loader to handle `.jsx` files

```
 ERROR in .-exercise-perseus-plugin/kolibri_exercise_perseus_plugin/~/perseus/src/widgets/simple-markdown-tester-editor.jsx
    Module parse failed: /home/christian/repos/kolibri-exercise-perseus-plugin/kolibri_exercise_perseus_plugin/node_modules/perseus/src/widgets/simple-markdown-tester-editor.jsx Unexpected token (17:15)
    You may need an appropriate loader to handle this file type.
```
etc..

OK. Been trying to figure out what's going on for a while now, and I've narrowed it down a bit. 

Looks like the custom webpack config in the plugin isn't being taken into account because our build process can't find the file.

For whatever reason, 'parseBundlePlugin''s use of `require(path.resolve(path.join(data.plugin_path, 'webpack.config.js')))` fails inside of the perseus plugin's `data.plugin_path`. It's especially weird because if I do an `fs.readdirSync(data.plugin_path)` on the perseus plugin, it gets listed fine. See output below:

```
$ yarn run build
yarn run v0.27.5
$ webpack --config ./frontend_build/src/webpack.config.prod.js
INFO     Writing webpack_json output to /tmp/117821-16561-1pputml.xburgm0a4i.json
INFO:kolibri.core.webpack.management.commands.webpack_json:Writing webpack_json output to /tmp/117821-16561-1pputml.xburgm0a4i.json
Displaying info for: /home/david/Work/kolibri/kolibri/plugins/management
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'assets',
  'build',
  'hooks.py',
  'kolibri_plugin.py',
  'static',
  'templates',
  'templatetags',
  'urls.py',
  'views.py' ]
Looking for:  /home/david/Work/kolibri/kolibri/plugins/management/webpack.config.js
Failed with: 
 { Error: Cannot find module '/home/david/Work/kolibri/kolibri/plugins/management/webpack.config.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at parseBundlePlugin (/home/david/Work/kolibri/frontend_build/src/parse_bundle_plugin.js:62:20)
    at readBundlePlugins (/home/david/Work/kolibri/frontend_build/src/read_bundle_plugins.js:56:18)
    at Object.<anonymous> (/home/david/Work/kolibri/frontend_build/src/webpack.config.js:19:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32) code: 'MODULE_NOT_FOUND' } 
 No local config


Displaying info for: /home/david/Work/kolibri/kolibri/plugins/setup_wizard
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'assets',
  'build',
  'hooks.py',
  'kolibri_plugin.py',
  'middleware.py',
  'static',
  'templates',
  'templatetags',
  'urls.py',
  'views.py' ]
Looking for:  /home/david/Work/kolibri/kolibri/plugins/setup_wizard/webpack.config.js
Failed with: 
 { Error: Cannot find module '/home/david/Work/kolibri/kolibri/plugins/setup_wizard/webpack.config.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at parseBundlePlugin (/home/david/Work/kolibri/frontend_build/src/parse_bundle_plugin.js:62:20)
    at readBundlePlugins (/home/david/Work/kolibri/frontend_build/src/read_bundle_plugins.js:56:18)
    at Object.<anonymous> (/home/david/Work/kolibri/frontend_build/src/webpack.config.js:19:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32) code: 'MODULE_NOT_FOUND' } 
 No local config


Displaying info for: /home/david/Work/kolibri/kolibri/plugins/management
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'assets',
  'build',
  'hooks.py',
  'kolibri_plugin.py',
  'static',
  'templates',
  'templatetags',
  'urls.py',
  'views.py' ]
Looking for:  /home/david/Work/kolibri/kolibri/plugins/management/webpack.config.js
Failed with: 
 { Error: Cannot find module '/home/david/Work/kolibri/kolibri/plugins/management/webpack.config.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at parseBundlePlugin (/home/david/Work/kolibri/frontend_build/src/parse_bundle_plugin.js:62:20)
    at readBundlePlugins (/home/david/Work/kolibri/frontend_build/src/read_bundle_plugins.js:56:18)
    at Object.<anonymous> (/home/david/Work/kolibri/frontend_build/src/webpack.config.js:19:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32) code: 'MODULE_NOT_FOUND' } 
 No local config


Displaying info for: /home/david/Work/kolibri/kolibri/plugins/learn
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'assets',
  'build',
  'hooks.py',
  'kolibri_plugin.py',
  'node_modules',
  'static',
  'templates',
  'templatetags',
  'urls.py',
  'views.py' ]
Looking for:  /home/david/Work/kolibri/kolibri/plugins/learn/webpack.config.js
Failed with: 
 { Error: Cannot find module '/home/david/Work/kolibri/kolibri/plugins/learn/webpack.config.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at parseBundlePlugin (/home/david/Work/kolibri/frontend_build/src/parse_bundle_plugin.js:62:20)
    at readBundlePlugins (/home/david/Work/kolibri/frontend_build/src/read_bundle_plugins.js:56:18)
    at Object.<anonymous> (/home/david/Work/kolibri/frontend_build/src/webpack.config.js:19:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32) code: 'MODULE_NOT_FOUND' } 
 No local config


Displaying info for: /home/david/Work/kolibri/kolibri/plugins/style_guide
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'assets',
  'build',
  'hooks.py',
  'kolibri_plugin.py',
  'node_modules',
  'package.json',
  'static',
  'templates',
  'templatetags',
  'urls.py',
  'views.py',
  'vue-doc-loader.js',
  'webpack.config.js',
  'yarn.lock' ]
Looking for:  /home/david/Work/kolibri/kolibri/plugins/style_guide/webpack.config.js
Found! Local Config is: 
 { resolveLoader: { alias: { 'vue-doc': '/home/david/Work/kolibri/kolibri/plugins/style_guide/vue-doc-loader' } } }


Displaying info for: /home/david/Work/perseus-plugin/kolibri_exercise_perseus_plugin
Path looks like: /n [ '.eslintrc.js',
  '__init__.py',
  '__pycache__',
  'assets',
  'build',
  'content_types.json',
  'extractPerseusMessages.js',
  'generateKATranslations.js',
  'getMessages.js',
  'gettextToICU.js',
  'kolibri_plugin.py',
  'locale',
  'node_modules',
  'npm-debug.log',
  'package.json',
  'static',
  'webpack.config.js',
  'yarn.lock' ]
Looking for:  /home/david/Work/perseus-plugin/kolibri_exercise_perseus_plugin/webpack.config.js
Failed with: 
 { Error: Cannot find module 'webpack'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/david/Work/perseus-plugin/kolibri_exercise_perseus_plugin/webpack.config.js:6:15)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3) code: 'MODULE_NOT_FOUND' } 
 No local config


Displaying info for: /home/david/Work/kolibri/kolibri/plugins/media_player
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'assets',
  'build',
  'content_types.json',
  'kolibri_plugin.py',
  'node_modules',
  'package.json',
  'static',
  'yarn.lock' ]
Looking for:  /home/david/Work/kolibri/kolibri/plugins/media_player/webpack.config.js
Failed with: 
 { Error: Cannot find module '/home/david/Work/kolibri/kolibri/plugins/media_player/webpack.config.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at parseBundlePlugin (/home/david/Work/kolibri/frontend_build/src/parse_bundle_plugin.js:62:20)
    at readBundlePlugins (/home/david/Work/kolibri/frontend_build/src/read_bundle_plugins.js:56:18)
    at Object.<anonymous> (/home/david/Work/kolibri/frontend_build/src/webpack.config.js:19:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32) code: 'MODULE_NOT_FOUND' } 
 No local config


Displaying info for: /home/david/Work/kolibri/kolibri/plugins/user
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'assets',
  'build',
  'hooks.py',
  'kolibri_plugin.py',
  'node_modules',
  'package.json',
  'static',
  'templates',
  'templatetags',
  'urls.py',
  'views.py',
  'yarn.lock' ]
Looking for:  /home/david/Work/kolibri/kolibri/plugins/user/webpack.config.js
Failed with: 
 { Error: Cannot find module '/home/david/Work/kolibri/kolibri/plugins/user/webpack.config.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at parseBundlePlugin (/home/david/Work/kolibri/frontend_build/src/parse_bundle_plugin.js:62:20)
    at readBundlePlugins (/home/david/Work/kolibri/frontend_build/src/read_bundle_plugins.js:56:18)
    at Object.<anonymous> (/home/david/Work/kolibri/frontend_build/src/webpack.config.js:19:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32) code: 'MODULE_NOT_FOUND' } 
 No local config


Displaying info for: /home/david/Work/kolibri/kolibri/plugins/coach
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'api.py',
  'api_urls.py',
  'assets',
  'build',
  'hooks.py',
  'kolibri_plugin.py',
  'node_modules',
  'package.json',
  'serializers.py',
  'static',
  'templates',
  'templatetags',
  'test',
  'urls.py',
  'utils',
  'views.py',
  'yarn.lock' ]
Looking for:  /home/david/Work/kolibri/kolibri/plugins/coach/webpack.config.js
Failed with: 
 { Error: Cannot find module '/home/david/Work/kolibri/kolibri/plugins/coach/webpack.config.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at parseBundlePlugin (/home/david/Work/kolibri/frontend_build/src/parse_bundle_plugin.js:62:20)
    at readBundlePlugins (/home/david/Work/kolibri/frontend_build/src/read_bundle_plugins.js:56:18)
    at Object.<anonymous> (/home/david/Work/kolibri/frontend_build/src/webpack.config.js:19:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32) code: 'MODULE_NOT_FOUND' } 
 No local config


Displaying info for: /home/david/Work/kolibri/kolibri/plugins/html5_app_renderer
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'assets',
  'build',
  'content_types.json',
  'kolibri_plugin.py',
  'node_modules',
  'package.json',
  'static',
  'yarn.lock' ]
Looking for:  /home/david/Work/kolibri/kolibri/plugins/html5_app_renderer/webpack.config.js
Failed with: 
 { Error: Cannot find module '/home/david/Work/kolibri/kolibri/plugins/html5_app_renderer/webpack.config.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at parseBundlePlugin (/home/david/Work/kolibri/frontend_build/src/parse_bundle_plugin.js:62:20)
    at readBundlePlugins (/home/david/Work/kolibri/frontend_build/src/read_bundle_plugins.js:56:18)
    at Object.<anonymous> (/home/david/Work/kolibri/frontend_build/src/webpack.config.js:19:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32) code: 'MODULE_NOT_FOUND' } 
 No local config


Displaying info for: /home/david/Work/kolibri/kolibri/core
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'api.py',
  'apps.py',
  'assets',
  'build',
  'context_processors',
  'device',
  'discovery',
  'errors.py',
  'exams',
  'fields.py',
  'hooks.py',
  'kolibri_plugin.py',
  'models.py',
  'serializers.py',
  'settings.py',
  'static',
  'templates',
  'templatetags',
  'test',
  'urls.py',
  'views.py',
  'webpack' ]
Looking for:  /home/david/Work/kolibri/kolibri/core/webpack.config.js
Failed with: 
 { Error: Cannot find module '/home/david/Work/kolibri/kolibri/core/webpack.config.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at parseBundlePlugin (/home/david/Work/kolibri/frontend_build/src/parse_bundle_plugin.js:62:20)
    at readBundlePlugins (/home/david/Work/kolibri/frontend_build/src/read_bundle_plugins.js:56:18)
    at Object.<anonymous> (/home/david/Work/kolibri/frontend_build/src/webpack.config.js:19:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32) code: 'MODULE_NOT_FOUND' } 
 No local config


Displaying info for: /home/david/Work/kolibri/kolibri/plugins/document_pdf_render
Path looks like: /n [ '__init__.py',
  '__pycache__',
  'assets',
  'build',
  'content_types.json',
  'kolibri_plugin.py',
  'node_modules',
  'package.json',
  'static',
  'webpack.config.js',
  'yarn.lock' ]
Looking for:  /home/david/Work/kolibri/kolibri/plugins/document_pdf_render/webpack.config.js
Found! Local Config is: 
 { entry: { pdfJSWorker: 'pdfjs-dist/build/pdf.worker.entry' } }
```

I'm pretty stumped. Ideas? @rtibbles

--- a/kolibri_exercise_perseus_plugin/webpack.config.js
+++ b/kolibri_exercise_perseus_plugin/webpack.config.js
@@ -3,7 +3,7 @@
  * It will be bundled into the webpack configuration at build time.
  */
 var path = require('path');
-var webpack = require('webpack');
+var webpack = require('perseus/node_modules/webpack');
 
 module.exports = {
   resolve: {

